ratlib/autocorrect.py

```diff
@@ -1,7 +1,7 @@
 """
 System name autocorrection
 
-Copyright (c) 2017 The Fuel Rats Mischief, 
+Copyright (c) 2017 The Fuel Rats Mischief,
```

Comment:
unnecessary modification.

Reply:
This is mecha2. I think you may be applying project standards from Mecha3 to a rather simple hotfix for Marenthyu's project here...

Reply:
It's whatever. Yes, unnecessary, but who cares, really?